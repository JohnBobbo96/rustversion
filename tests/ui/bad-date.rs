#[rustversion::nightly(stable)]
struct S;

fn main() {}
