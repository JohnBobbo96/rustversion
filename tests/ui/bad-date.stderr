error: expected nightly date, like 2020-10-26
 --> $DIR/bad-date.rs:1:24
  |
1 | #[rustversion::nightly(stable)]
  |                        ^^^^^^
